R/build-articles.R

```diff
@@ -200,7 +200,10 @@ build_article <- function(name,
       options <- list()
     }
   } else {
-    format <- build_rmarkdown_format(pkg, depth = depth, data = data, toc = TRUE)
+    format <- build_rmarkdown_format(pkg, "article",
```

Comment:
Can you please use tidyverse style here? (i.e. https://style.tidyverse.org/syntax.html#long-lines)

Reply:
Ah yes. Done and pushed.